src/commands/environments/create.ts

```diff
@@ -84,12 +85,16 @@ export default class EnvironmentCreate extends Command {
       const { data: environment } = await this.app.api.post(`/accounts/${account_id}/environments`, data);
       return environment;
     } catch (err) {
+      //TODO:89:we shouldn't have to do this on the client side
```

Comment:
Agreed these should all be responses from the api and have only one error here or a global handler.